javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.gyb

```diff
@@ -0,0 +1,630 @@
+%{
+## ArraysJDK9Methods.scala.gyb
+##
+##
+## To generate this file's output manually, execute the python script
+## 'scripts/gyb.py' under the project root. For example, from the project root:
+##
+##   scripts/gyb.py \
+##     javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.gyb \
+##     --line-directive '' \
+##     -o  javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.template
```

Comment:
If we'd follow the suggestion bellow we should add this file to `scripts/gyb_all.sh`.

Reply:
Done

Reply:
Done